platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -58,6 +66,9 @@
   private static final String USERS_PROPERTIES_FILE =
       new AbsolutePathResolver("etc/users.properties").getPath();
 
+  static final Path USERS_PROPERTIES_FILE_PATH =
```

Comment:
✏️ I believe this is package-private only for testing purposes, correct? If so, I recommend it be annotated with the `@VisibleForTesting` annotation.